plugin/rewrite/README.md

```diff
@@ -206,17 +206,13 @@ rewrites the first local option with code 0xffee, setting the data to "abcd". Eq
 }
 ~~~
 
-* A variable data is specified with a pair of curly brackets `{}`. Following are the supported variables by default:
+* A variable data is specified with a pair of curly brackets `{}`. Following are the supported variables:
   {qname}, {qtype}, {client_ip}, {client_port}, {protocol}, {server_ip}, {server_port}.
-Any plugin that can provide it's own additional variables by implementing metadata.Provider interface. If you are going to use metadata variables then metadata plugin must be enabled.
 
 Example:
 
-~~~ corefile
-. {
-    metadata
-    rewrite edns0 local set 0xffee {client_ip}
```

Comment:
At the end, we will want with "metadata" (because we expect to be able to add some of the metadata in the edns0 records).

Can be in second time, once this refactoring is in.